tensorflow/compiler/xla/client/lib/math.cc

```diff
@@ -341,7 +341,10 @@ XlaOp ErfInv(XlaOp x) {
       0.00943887047f,   1.00167406f,     2.83297682f};
 
   auto one = ScalarLike(x, 1.0);
-  auto w = -Log((one - x) * (one + x));
+  //Computes logarithm of (1+arg). This function is more precise
```

Comment:
Would you please change "Computes" to "Compute"?  See the comment at the end of this routine "Handle edge cases".

Reply:
Thanks @bixia1.